app.js

```diff
@@ -6,85 +6,26 @@ console.log("var = travel asks the user if I have ever lived abroad");
 console.log("var = animals asks the user if I like animals");
 console.log("var = people asks the user if I like people");
 
-function gamesQuestion() {
-  var games = prompt('Do I love to play video games? (answer yes or no)');
+function askQuestion(question, yesAnswer, noAnswer) {
+  var response = prompt(question);
 
-  if (games == null) {
+  if (response == null) {
     alert('Canceled');
-  }else if (games.trim().toLowerCase() == 'yes' || games.trim().toLowerCase() == 'y' ){
-    alert('You are correct!');
-  }else if (games.trim().toLowerCase() == 'no' || games.trim().toLowerCase() == 'n' ){
-    alert('I play video games just about everyday!');
+  }else if (response.trim().toLowerCase() == 'yes' || response.trim().toLowerCase() == 'y' ){
+    alert(yesAnswer);
+  }else if (response.trim().toLowerCase() == 'no' || response.trim().toLowerCase() == 'n' ){
+    alert(noAnswer);
   }else{
     alert("Your response wasn't yes or no!");
   }
-  return games;
+  return response;
 }
-var games = gamesQuestion();
+var games = askQuestion('Do I love to play video games? (answer yes or no)', 'You are correct!', 'I play video games just about everyday!');
```

Comment:
for the second stretch-goal, you need to iterate over arrays to get the question properties. you can't call each of them individually.